### PR TITLE
Add heading level and alignment to inspector

### PR DESCRIPTION
**Pull request by ellatrix**
Just an idea. See #751.

This PR removes levels 1, 4 and 5 from the inline toolbar, and adds levels 1-6 to the inspector. Also adds alignment.

@joen Feel free to style any way you want.

### CODE REVIEW

**Comment by jasmussen** [comment]
This is really really interesting. I can't quite yet tell whether I like it, but I think I love it. But it feels like we need more opinions on this one. CC: @melchoyce @mtias @folletto @afercia 

Here's a screenshot:

screen shot 2017-06-23 at 09 55 28

**Comment by jasmussen** [comment]
(Also, I still have on my todo to make better icons for the H levels, but I'll do that separately)

**Comment by afercia** [comment]
Gut reaction: I like it! Definitely something worth exploring, also some user testing would be nice.
Just a small thing, it would be great to don't skip heading levels, currently there's a jump from h2 to h4:

screen shot 2017-06-23 at 10 00 57

**Comment by melchoyce** [comment]
I have a similar positive gut reaction!

**Comment by jasmussen** [comment]
Given the positive reactions, let's get this in master and test it.

File: components/toolbar/style.scss [hunk 1]
```diff
@@ -36,6 +36,11 @@
 			height: $icon-button-size - 16px;
 		}
 	}
+
+	.editor-sidebar & {
```
Comment (mtias) [inline components/toolbar/style.scss]: I prefer to include these styles in `editor-sidebar` instead.

Comment (iseulde) [inline components/toolbar/style.scss]: cc @jasmussen

Comment (jasmussen) [inline components/toolbar/style.scss]: 👍 👍

**Comment by mtias** [comment]
I'm a bit conflicted with moving the alignment, because now some blocks will have alignment in quick access, and some will have it in the inspector.

**Comment by iseulde** [comment]
@mtias Yeah, I wonder if we should move it to the inspector everywhere, as an advanced setting.

**Comment by jasmussen** [comment]
Matías makes a good point.

Should we show instead `Switcher | H2-4 | Alignments`

.. and then the rest in the inspector?

Instead of what it is now, `Switcher | H2-4 | Inline`?

**Comment by iseulde** [comment]
I don't mind that, or show alignment in the inspector for all blocks. An argument against the latter is that centre alignment is used a lot.

Something in between we should explore: centre alignment toggle for all blocks (one button), and more alignment options in the inspector.

N.b.: alignment has nothing to do with LTR and RTL. I saw some confusion a few months ago.

**Comment by mtias** [comment]
I think I'd rather see the switcher in the inspector, to be honest. The other part of the conversation is that we've said the inspector is for advanced settings, and the design optimizes for that—hidden on mobile, etc. Specially if it become a modal, it would be awkward to have alignment there because you cannot see what you change with a modal obscuring it.

**Comment by iseulde** [comment]
I'd also rather see it in the inspector. Regarding the mobile argument, the same could be said about all other advanced controls as well, e.g. drop cap. Not sure how alignment would be different.

**Comment by jasmussen** [comment]
Part of the "basic & advanced split" conversation is also the implicit tradeoff that we are okay with it not being as accessible on mobile. 

That said, Google Docs does something interesting on mobile:

screenshot_20170626-150919

We could presumably do something similar with our inspector, but it could only be for block level actions, as inline actions would invoke the soft keyboard.

**Pull request merged**